OpenRA.Mods.Common/Widgets/LabelWidget.cs

```diff
@@ -28,15 +29,40 @@ public class LabelWidget : Widget
 		public bool Contrast = ChromeMetrics.Get<bool>("TextContrast");
 		public Color ContrastColor = ChromeMetrics.Get<Color>("TextContrastColor");
 		public bool WordWrap = false;
+		public int LineSpacing = 2;
+
+		[Desc("line spacing = LineSpacing * font size / LineSpacingDivisor or LineSpacing if this is 0.")]
```

Comment:
What is the benefit of this complexity vs having a single `[Desc("Line spacing as a multiple of the line height")]float LineSpacing = 1.2; // or whatever`?